lte/gateway/python/magma/mobilityd/BUILD.bazel

```diff
@@ -108,3 +108,9 @@ py_library(
     visibility = ["//visibility:public"],
     deps = ["//lte/protos:keyval_python_proto"],
 )
+
+py_library(
+    name = "mac",
+    srcs = ["mac.py"],
```

Comment:
This source file is already provided in the `dhcp_desc` library, could we just use that instead of creating a new one? Or does it make sense to split them up in general?

Reply:
We changed this as you suggested.